src/lib/components/Widget.js

```diff
@@ -67,11 +67,11 @@ class Widget extends React.Component {
             <img src={Logo} alt="SSW Logo" height="60" width="130"></img>
           </a>
         ) : ''}
-          <h1>
+          <h4>
             <a rel="noreferrer" target="_blank" href={`${this.sswUrl}/rules`}>
```

Comment:
Might need to add a check to make sure that this doesn't link to the same page. (If this is on /rules it shouldn't link to /rules) as per https://www.ssw.com.au/rules/do-you-avoid-linking-a-page-to-itself

Reply:
That should be ok because the rules widget only shows on pages that it can't link to (i.e People/Category)